src/ipvs/ip_vs_proto_udp.c

```diff
@@ -177,8 +181,8 @@ static int udp_conn_sched(struct dp_vs_proto *proto,
     if ((*conn)->dest->fwdmode == DPVS_FWD_MODE_FNAT && g_uoa_max_trail > 0) {
         struct conn_uoa *uoa;
 
-        (*conn)->prot_data = rte_zmalloc(NULL, sizeof(struct conn_uoa), 0);
-        if (!(*conn)->prot_data) {
+        err = rte_mempool_get(this_uoa_cache, (void **)&((*conn)->prot_data));
```

Comment:
for now, struct conn_uoa has 6 bytes only, maybe use prot_data directly
uoa = (void*)&(*conn)->prot_data